server.js

```diff
@@ -56,72 +57,37 @@ app.get('/itemInformation/:itemId', (req, res) => {
 
   if (itemId.includes('array')) {
     const itemsInArray = itemId.substring(5);
-    const itemIds = itemsInArray.split(',');
-    const invalidId = false;
-
-    for (var i = 0; i < itemIds.length; i++) {
-      if (itemIds[i] < 100 || itemIds[i] > 199) {
-        res.status(404).send('Invalid itemId');
-        invalidId = true;
-        break;
-      }
-    }
-
-    if (!invalidId) {
-      db.getTitlesAndBrands(itemIds)
-        .then((data) => {
-          res.send(data);
-        })
-        .catch((err) => {
-          res.status(404).send('error in getTitlesAndBrands: ', err);
-        });
-    }
-  } else if (itemId < 100 || itemId > 199) {
-    console.log(itemId);
-    res.status(404).send('Invalid itemId');
-  } else {
-    db.getTitleAndBrand(itemId)
+    const itemIds = itemsInArray.split(',').map((id) => Number(id));
```

Comment:
You convert `itemId` to int here and in line 75, but don't convert it in lines 112, 124.